ext/drreg/drreg.c

```diff
@@ -1948,13 +2893,21 @@ drreg_init(drreg_options_t *ops_in)
         ops.error_callback = ops_in->error_callback;
 
     if (prior_slots > 0) {
-        if (!dr_raw_tls_cfree(tls_slot_offs, prior_slots))
+        if (!dr_raw_tls_cfree(tls_simd_offs, prior_slots + 1))
```

Comment:
I think I missed a comment here, but I think it is work adding an inline comment near the prior_slots + 1 to explain the magic value. Smthg like /* for indirect spills */